module/VuFind/tests/integration-tests/src/VuFindTest/Mink/RecordActionsTest.php

```diff
@@ -946,6 +946,82 @@ function () use ($page, $checked) {
         $this->clickCss($page, '.logoutOptions a.logout');
     }
 
+    /**
+     * Test adding comments, rating and tag to a record and then
+     * delete them from user account menu
+     *
+     * @return void
+     */
+    public function testUserContentDeletion()
+    {
+        $page = $this->gotoRecord();
+        $this->changeConfigs(
+            [
+                'config' => [
+                    'Social' => [
+                        'rating' => true,
+                        'remove_rating' => true,
+                    ],
+                ],
+            ]
+        );
+
+        $this->clickCss($page, '.record-tabs .usercomments a');
+        $this->findCss($page, '.comment-form');
+        $this->openCommentsLoginModal($page);
+        $this->fillInLoginForm($page, 'username1', 'test');
+        $this->submitLoginForm($page);
+
+        $page = $this->gotoRecord();
```

Comment:
Do you need this extra "goToRecord" call? Might it only have been needed in the past because of the configuration order issue I mentioned above?